src/client_lib/pegasus_scanner_impl.cpp

```diff
@@ -156,6 +161,11 @@ void pegasus_client_impl::pegasus_scanner_impl::_async_next_internal()
         std::string hash_key, sort_key;
         pegasus_restore_key(_kvs[_p].key, hash_key, sort_key);
         std::string value(_kvs[_p].value.data(), _kvs[_p].value.length());
+        int ttl_seconds = 0;
+        if (_kvs[_p].__isset.expire_ts_seconds) {
+            auto expire_ts_seconds = static_cast<uint32_t>(_kvs[_p].expire_ts_seconds);
+            ttl_seconds = static_cast<int>(expire_ts_seconds - utils::epoch_now());
```

Comment:
We should prevent a corner case that `ttl_seconds` overflowed. Because it's a RPC, means it is possible to cost some seconds to get scan response( or there is clock drift), suppose `expire_ts_seconds` is `1600000000`, `epoch_now` on server side is `1599999999`, so it's not expired and will return to client. But on client side `epoch_now` is `1600000001`, then `ttl_seconds` is overflowed and will set this kv a large ttl in callback which is not what we expect.